charts/super-agent/values.yaml

```diff
@@ -0,0 +1,181 @@
+    valuesFrom:
+
+      # - kind: ConfigMap or Secret
+      #   name: Name of the values referent, in the same namespace as the HelmRelease.
+      #   valuesKey (Optional): The data key where the values.yaml or a specific value can be found. 
+      #                         Defaults to `values.yaml` when omitted.
+      #   targetPath (Optional): The YAML dot notation path at which the value should be merged. When set, the 
+      #                          valuesKey is expected to be a single flat value. Defaults to `nil` when omitted, which
+      #                          results in the values getting merged at the root.
+
+      # Examples:
+      # - kind: ConfigMap
+      #   name: prod-env-values
+      #   valuesKey: values-prod.yaml
+      # - kind: Secret
+      #   name: prod-tls-values
+      #   valuesKey: crt
+      #   targetPath: tls.crt
+
+    # -- Change the behavior of the operator while installing the chart for the first time. This should only be changed
+    # by advanced users that know what they are doing. Exposes the remediations that the operator is going to try
+    # before give up installing the chart in case it hits an error.
+    # @default -- See `values.yaml`
+    install:
+      crds: CreateReplace  # New CRDs are created, existing CRDs are updated (replaced) but not deleted.
+      replace: true  # Tells the Helm install action to re-use the ‘ReleaseName’, but only if that name is a deleted release which remains in the history.
+      remediation:
+        retries: 3  # Retries is the number of retries that should be attempted on failures before givin up. An uninstall is performed between each attempt.
+
+    # -- Change the behavior of the operator while upgrading the chart. This should only be changed by advanced users
+    # that know what they are doing. Exposes the remediations that the operator is going to try before give up
+    # installing the chart in case it hits an error.
+    # @default -- See `values.yaml`
+    upgrade:
+      crds: CreateReplace  # New CRDs are created, existing CRDs are updated (replaced) but not deleted.
+      force: true  # Force forces resource updates through a replacement strategy.
+      cleanupOnFail: true  # Allows deletion of new resources created during the Helm upgrade action when it fails.
+      remediation:
+        retries: 3  # Retries is the number of retries that should be attempted on failures before givin up. An uninstall is performed between each attempt.
+        strategy: rollback  # rollback or uninstall.
+
+# -- Values for the Flux chat. Ref.: https://github.com/fluxcd-community/helm-charts/blob/flux2-2.10.2/charts/flux2/values.yaml
+# @default -- See `values.yaml`
+flux2:
+  # -- Enable or disable FluxCD completely. New Relic' Super Agent need Flux to work, but the user can use an already
+  # existing Flux deployment. With that use case, the use can disable Flux and use this chart to only install the CRs
+  # to deploy the Super Agent.
```

Comment:
```suggestion
  # -- Enable or disable FluxCD instasllation. New Relic' Super Agent need Flux to work, but the user can use an already
  # existing Flux deployment. When disabled, this chart can be used to install only the CRs needed to deploy the Super Agent.
```

Reply:
It is probably out of the scope of this PR and maybe it should be somewhere else but, should we describe the 'already existing Flux' requirements to deploy the Super Agent?

Maybe this values file is can be considered self-documentation, if that is the case we could say it explicitly.

Reply:
I agree, that we have to also pin the version and capabilities. But I think it is too soon as everything can change. We can do it at the release of the MVP.

If you have an idea of something that we can start defining right now that can be written in stone?